lib/chef/application/knife.rb

```diff
@@ -216,6 +216,15 @@ def print_help_and_exit(exitcode = 1, fatal_message = nil)
     rescue OptionParser::InvalidOption => e
       puts "#{e}\n"
     end
+
+    if want_help?
+      puts "#{Chef::Dist::PRODUCT}: #{Chef::VERSION}"
```

Comment:
`knife -v` vs `Chef::VERSION` let me know if any consequences of this?

Reply:
VERSION seems fine here. That's what's actually loaded